modules/using-assisted-installer-oci-agent-iso.adoc

```diff
@@ -0,0 +1,47 @@
+
+.Prerequisites
+
+* You created a child compartment and an object storage bucket on {oci}. See the "Creating OCI resources and services" section.
+* You reviewed details about the {product-title} xref:../../architecture/architecture-installation.adoc#architecture-installation[installation and update] processes. 
+* You completed the link:https://docs.google.com/forms/d/e/1FAIpQLSdQHmw0BUWjQxPKC5_G7lk2_o9Pcnn2ON84Al6s2Ir254gv6Q/viewform[Request Access to Red Hat OpenShift on Oracle Cloud Infrastructure in Developer Preview] form.
+* If you use a firewall and you plan to use a Telemetry service, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured your firewall], so that {product-title} can access the sites required.
+* Before you create a virtual machines (VM), refer to {op-system-base-full} certified shapes, instance types, to identify the supported {oci} VM shapes. See link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types] on Red Hat Ecosystem Catalog portal.
+
+.Procedure
+
+. From the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[**Install OpenShift with the Assisted Installer**] page on the Hybrid Cloud Console, generate the discovery ISO image by completing all the required Assisted Installer steps. 
++
+.. For the **Cluster Details** step, you complete the following fields:
++
+... **Cluster name**: Specify the name of your cluster, such as `ocidemo`. 
+... **Base domain**: Specify the base domain of the cluster, such as `splat-oci.devcluster.openshift.com`. Provided you previously created a compartment on {oci}, you can get this information by going to **DNS management > Zones > List scope** and then select the parent compartment. Your base domain should show under the **Public zones** tab.
+... **OpenShift version**: Specify OpenShift 4.15 or a later version.
+... **CPU architecture**: Specify `x86_64` or `Arm64`.
+... **Integrate with external partner platforms**, where you must specify `Oracle Cloud Infrastructure`.
++
+After you specify this value, the *Include custom manifests* checkbox is automatically selected. 
++
+[IMPORTANT] 
+====
+You can keep the default settings for the **Operators** step.
+====
+
+. For the **Host Discovery** step, click on the **Add hosts** button to display a dialog box. For the **SSH public key** field, add your SSH key from your local system. Click the **Generate Discovery ISO** button to generate the discovery image ISO file. Ensure that you download the file to your local system.
++
+[TIP]
+====
+You can create an SSH authentication key pair by using the `ssh-keygen` tool. 
+====
+
+. On {oci}, go to **Resources > Objects** and then click the **Upload** button. Upload the Agent ISO image to the bucket, and then click the **Upload** button to complete the upload operation.
```

Comment:
I did not find Resources > Objects in current Oracle Cloud.
Should be: 
- Go to Storage -> Object Storage & Archive Storage.
- Find previously created Bucket and click 
- Click Upload button in "Objects" table

Reply:
Yes, it will better. Just mention the step "Upload the Agent ISO image to the bucket following the procedure " and link to Oracle docs. Maybe this is will be  better [https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Object/Tasks/managingobjects_topic-To_upload_objects_to_a_bucket.htm](url)

Reply:
Thanks. Updated to:

> Upload the Agent ISO image to the bucket by completing the steps in the link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Object/Tasks/managingobjects_topic-To_upload_objects_to_a_bucket.htm[Uploading an Object Storage Object to a Bucket] section of the Oracle documentation.